### PR TITLE
Rayfile

### PR DESCRIPTION
**Pull request by KrailUnknown**
No tests yet. Not sure. what to test, as everything is done by other services, that are already tested.

`bpm` package intentionally not used.

### CODE REVIEW

File: internal/domain/rayfile/file.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package rayfile
+
+type Rayfile struct {
+	Version   string     `toml:"version"`
+	Workspace *workspace `toml:"workspace"`
+}
+
+type workspace struct {
```
Comment (g10z3r) [inline internal/domain/rayfile/file.go:8]: I think this should be **public**, and for all included structs, use suffix `Def`, like `WorkspaceDef`

File: internal/domain/rayfile/service.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package rayfile
+
+import (
+	"io"
+)
+
+type tomlService interface {
```
Comment (g10z3r) [inline internal/domain/rayfile/service.go:7]: Some information on naming conventions. For such type of interfaces use name `[parentName]TOMLEncoder`, so will be `rayfileTOMLEncoder`

File: cli/command/conf.go
Comment (g10z3r) [inline cli/command/conf.go:1]: Well, this is just a tmp command file, right?

**Review by g10z3r — APPROVED** [review]

File: cli/command/conf.go
Comment (KrailUnknown) [reply cli/command/conf.go:1]: Yes, to be removed sometime later

File: internal/domain/rayfile/service.go (hunk 2 above)
Comment (KrailUnknown) [reply internal/domain/rayfile/service.go:7]: I think I got your point. Renamed it do `rayfileTOMLDecoder`

I also have `ioWrapper` there. Should I also change the name or is it okay?

Comment (g10z3r) [reply internal/domain/rayfile/service.go]: yeah, lets keep it with rayfile prefix for now

**Review by g10z3r — APPROVED** [review]

**Pull request merged**